hangman_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule HangmanTest do
+  use ExUnit.Case
+  doctest Hangman
+
+  test "Words are returned" do
```

Comment:
I'd like to give you bonus marks for this, but the don't compile (it should be Hangman.State)